{{cookiecutter.project_slug}}/notebooks/example.ipynb

```diff
@@ -4,7 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    {% if cookiecutter.package_manager == 'poetry' %}
+    "You can use your module `{{ cookiecutter.module_name }}` from here, provided you're running the jupyter server using `poetry run jupyter server`."
```

Comment:
in the readme you wrote `poetry run jupyter notebook`, here `poetry run jupyter server`.

Which one is correct?

Reply:
Sorry, my mistake: `poetry run jupyter notebook` is correct